packages/eslint-config-recommended/README.md

```diff
@@ -0,0 +1,52 @@
+# @shiftcode/eslint-config-recommended
+
+> 🎯 Target runtime: es2022 ([Node >= 18](https://node.green/#ES2022))
+
+> ⚠️ This module exports code using CommonJS
+
+This module provides an eslint default rule-set configuration for shiftcode projects.\
+( currently, it is verified against mt-fadas - further projects coming soon )
```

Comment:
remove this comment, since it refers to an internal project and the code here is publicly available